spec/services/legacy/dataset_import_service_spec.rb

```diff
@@ -53,18 +54,20 @@
     end
 
     it "creates the datafiles for the imported dataset" do
+      first_resource = legacy_dataset["resources"][0]
+      first_resource["last_modified_at"] = "2017-12-14T09:35:25.928982"
```

Comment:
Could we use `Time.now` here instead of a constant. I usually interpret a constant as something special, but I think this is just a throwaway value.

Reply:
Sure, I think that might have been a hangover from a previous implementation.

Reply:
Seems I forgot to push! Now done 👍